dandisets/dandiset_000008_converted.yaml

```diff
@@ -0,0 +1,115 @@
+contributor:
+- identifier: ''
+  name: Scala, Federico
+  roleName:
+  - DataCollector
+  - Author
+  - ContactPerson
+- identifier:
+    propertyID: ORCID
+    value: 0000-0002-5639-7209
+  name: Kobak, Dmitry
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0003-4458-117X
+  name: Bernabucci, Matteo
+  roleName:
+  - Author
+- name: Bernaerts, Yves
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0003-1963-8285
+  name: Cadwell, Cathryn Rene
+  roleName:
+  - Author
+- name: Castro, Jesus Ramon
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0002-4922-8781
+  name: Hartmanis, Leonard
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0001-8066-1383
+  name: Jiang, Xiaolong
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0001-9532-788X
+  name: Laturnus, Sophie
+  roleName:
+  - Author
+- name: Miranda, Elanine
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0001-8736-527X
+  name: Mulherkar, Shalaka
+  roleName:
+  - Author
+- name: Tan, Zheng Huan
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0002-9361-5607
+  name: Yao, Zizhen
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0002-0326-5878
+  name: Zeng, Hongkui
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0001-6473-1740
+  name: Sandberg, Rickard
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0002-0199-4727
+  name: Berens, Philipp
+  roleName:
+  - Author
+- identifier:
+    propertyID: ORCID
+    value: 0000-0002-4305-6376
+  name: Tolias, Andreas Savas
+  roleName:
+  - Author
+  - ContactPerson
+dandisetStats:
+- 1328
+- numberOfSubjects: 266
```

Comment:
Why is the `1328` here, and as a list? Per `models.py`, `dandisetStats` is supposed to contain an object (of type [`DandisetStat`](https://github.com/satra/dandi-core/blob/1df89802f19fff8bf797230db8faa7c7a5a44b7e/scripts/models.py#L157-L163)), not an array. A similar thing is done in the converted metadata for `000012`.